smithy-aws-protocol-tests/model/rpcV2/cbor-structs.smithy

```diff
@@ -19,26 +19,52 @@ use smithy.test#httpResponseTests
         }
         method: "POST",
         bodyMediaType: "application/cbor",
-        uri: "/service/aws.protocoltests.rpcv2Cbor.RpcV2Protocol/operation/SimpleScalarProperties",
+        uri: "/service/RpcV2Protocol/operation/SimpleScalarProperties",
         body: "v2lieXRlVmFsdWUFa2RvdWJsZVZhbHVl+z/+OVgQYk3TcWZhbHNlQm9vbGVhblZhbHVl9GpmbG9hdFZhbHVl+kDz989saW50ZWdlclZhbHVlGQEAaWxvbmdWYWx1ZRkmkWpzaG9ydFZhbHVlGSaqa3N0cmluZ1ZhbHVlZnNpbXBsZXB0cnVlQm9vbGVhblZhbHVl9f8="
         params: {
-            trueBooleanValue: true,
-            falseBooleanValue: false,
             byteValue: 5,
             doubleValue: 1.889,
+            falseBooleanValue: false,
             floatValue: 7.624,
             integerValue: 256,
+            longValue: 9873,
             shortValue: 9898,
+            stringValue: "simple",
+            trueBooleanValue: true
+        }
+    },
+    {
+        id: "RpcV2CborSimpleScalarPropertiesUsingIndefiniteLength",
```

Comment:
This test doesn't really make sense, how is the implementation supposed to know to encode this way?

Reply:
We _should_ have response tests that check that we can handle it either way. Not 100% on whether you added those in this patch.

Reply:
I have added "appliesTo: server" to this one. The test is to ensure that the server is capable of deserializing simple scalar properties encoded using a map with a definite length.